### ISSUE TITLE
are you interested in co-maintainership, or transferring this project?

### ISSUE THREAD

**Issue by supervacuo** [ambv/dj.choices#22]
Hi @ambv! thanks for all your work on this over the years.

I'm using `dj.choices` on a couple of projects and would love to help out (I'm sure other people are in the same boat of having to use their own forks), starting with:

 - merging or closing the existing PRs (particularly #20 and now #21)
 - ensure Travis is working (may be solved by Tox update in #21)
 - making a release on PyPI (re: #17)

Would you be happy to make me a co-maintainer so I can start working on this, is there any other way I can help, or would you prefer to totally hand over this project?

**Comment by ambv** [comment]
Yes, I'd be glad to have help with this as I'm no longer a heavy Django user myself. Thanks for help with the existing PRs.

**Comment by jackcbrown89** [comment]
Have you considered transferring ownership to [jazzband](https://jazzband.co/)?

**Comment by hongquan** [comment]
Could someone take a look: https://github.com/ambv/dj.choices/pull/24 ?
